src/commands/lambda/__tests__/flare.test.ts

```diff
@@ -1,21 +1,444 @@
+      const cli = makeCli()
+      const context = createMockContext()
+      const code = await cli.run(['lambda', 'flare', '-f', 'func', '-c', '123', '-e', 'test@test.com'], context as any)
+      expect(code).toBe(0)
+      const output = context.stdout.toString()
+      expect(output).toMatchSnapshot()
+    })
+
+    it('prints error when no API key in env variables', async () => {
+      process.env = {}
+      const cli = makeCli()
+      const context = createMockContext()
+      const code = await cli.run(
+        ['lambda', 'flare', '-f', 'func', '-r', 'us-west-2', '-c', '123', '-e', 'test@test.com'],
+        context as any
+      )
+      expect(code).toBe(1)
+      const output = context.stderr.toString()
+      expect(output).toMatchSnapshot()
+    })
+
+    it('uses API key ENV variable and runs as expected', async () => {
+      process.env = {}
+      process.env[CI_API_KEY_ENV_VAR] = MOCK_API_KEY
+      process.env[API_KEY_ENV_VAR] = undefined
+      const cli = makeCli()
+      const context = createMockContext()
+      let code = await cli.run(
+        ['lambda', 'flare', '-f', 'func', '-r', 'test-region', '-c', '123', '-e', 'test@test.com'],
+        context as any
+      )
+      expect(code).toBe(0)
+      let output = context.stdout.toString()
+      expect(output).toMatchSnapshot()
+
+      process.env[CI_API_KEY_ENV_VAR] = undefined
+      process.env[API_KEY_ENV_VAR] = MOCK_API_KEY
+      code = await cli.run(
+        ['lambda', 'flare', '-f', 'func', '-r', 'test-region', '-c', '123', '-e', 'test@test.com'],
+        context as any
+      )
+      expect(code).toBe(0)
+      output = context.stdout.toString()
+      expect(output).toMatchSnapshot()
+    })
+
+    it('prints error when no case ID specified', async () => {
+      const cli = makeCli()
+      const context = createMockContext()
+      const code = await cli.run(
+        ['lambda', 'flare', '-f', 'func', '-r', 'us-west-2', '-e', 'test@test.com'],
+        context as any
+      )
+      expect(code).toBe(1)
+      const output = context.stderr.toString()
+      expect(output).toMatchSnapshot()
+    })
+
+    it('prints error when no email specified', async () => {
+      const cli = makeCli()
+      const context = createMockContext()
+      const code = await cli.run(['lambda', 'flare', '-f', 'func', '-r', 'us-west-2', '-c', '123'], context as any)
+      expect(code).toBe(1)
+      const output = context.stderr.toString()
+      expect(output).toMatchSnapshot()
+    })
+
+    it('runs successfully with all required options specified', async () => {
+      const cli = makeCli()
+      const context = createMockContext()
+      const code = await cli.run(VALID_INPUT, context as any)
+      expect(code).toBe(0)
+      const output = context.stdout.toString()
+      expect(output).toMatchSnapshot()
+    })
+  })
+
+  describe('writeFile', () => {
+    const MOCK_DATA = 'mock data'
+    const instance = new LambdaFlareCommand()
```

Comment:
An instance of the Flare command is being created just to use the context. But there is already a function that mocks the context.

You have previously used this context, why don't we follow the same approach here?

Reply:
Resoled with `const context = createMockContext() as BaseContext`